test/src/specs/tx_pool/send_secp_tx.rs

```diff
@@ -66,15 +66,30 @@ impl Spec for SendSecpTxUseDepGroup {
             .build();
 
         let tx_hash = tx.hash();
-        let message = H256::from(blake2b_256(tx_hash.as_slice()));
+        let witness = WitnessArgs::new_builder()
+            .lock(Bytes::from(vec![0u8; 65]).pack())
+            .build();
+        let witness_len = witness.as_bytes().len() as u64;
```

Comment:
Please change all `witness.as_bytes().len()` to:
```suggestion
        let witness_len = witness.as_slice().len() as u64;
```